core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -602,8 +623,9 @@ public void testExpireOlderThanMultipleCalls() {
     }
 
     // Retain last 2 snapshots and expire older than t3
-    table
-        .expireSnapshots()
+    RemoveSnapshots removeSnapshots = (RemoveSnapshots) table.expireSnapshots();
+    removeSnapshots
+        .withIncrementalCleanup(incrementalCleanup)
```

Comment:
Why doesn't this use `removeSnapshots(Table)`?

Reply:
Fixed